### PR TITLE
Update README promise example

### PR DESCRIPTION
**Pull request by renarsvilnis**
Hey,
I started learning Rabbitmq in node using this library and noticed that the readme promise example wasn't clear enough and didn't follow current node.js tendencies.

I updated the example, now It's way more readable and easier to understand what is going on for someone starting out.

### CODE REVIEW

**Comment by squaremo** [comment]
Thank you for this update to the example -- I guess it is past time to adopt more modern JavaScript syntax.

File: README.md [hunk 1]
```diff
@@ -74,35 +74,33 @@ require('amqplib/callback_api')
...
-      if (msg !== null) {
-        console.log(msg.content.toString());
-        ch.ack(msg);
-      }
-    });
-  });
-}).catch(console.warn);
+  // Sender
+  setInterval(async () => {
+    await ch.sendToQueue(queue, Buffer.from('something to do'));
```
Comment (squaremo) [inline README.md]: No need to `async/await` here, the return value is not a promise

**Comment by renarsvilnis** [comment]
Hi @squaremo,
Add the changes you wanted, also simplified other examples by moving to arrow function.

**Comment by kslr** [comment]
Why is there no progress?

**Pull request merged**